targets/unity-v2/source/Authentication/Strategies/EmailPasswordAuthStrategy.cs

```diff
@@ -0,0 +1,69 @@
+using System;
+using PlayFab.ClientModels;
+using UnityEngine;
+
+namespace PlayFab.Authentication.Strategies
+{
+    internal sealed class EmailPasswordAuthStrategy : IAuthenticationStrategy
```

Comment:
Why is this duplicating/hard-coding in a bunch of customId silent auth stuff, instead of just invoking the customId strategy?
(duplication of code, and code maintenance)